spec/lib/mvi/service_spec.rb

```diff
@@ -81,7 +81,7 @@
         VCR.use_cassette('mvi/find_candidate/valid_vet360_id') do
           response = subject.find_profile(user)
           expect(response.status).to eq('OK')
-          expect(response.profile['vet360_id']).to eq('123456789')
+          expect(response.profile['vet360_id']).to eq('1000')
```

Comment:
Vet360 ID is parsed [here](https://github.com/department-of-veterans-affairs/vets-api/blob/90970fd0e4504b66e70358bee56fc49309d13f59/lib/mvi/responses/id_parser.rb#L34).
`vet360_id: select_ids(select_extension(ids, /^\w+\^PI\^200VETS\^USDVA\^\w+$/, CORRELATION_ROOT_ID))&.first`

You can see that is just taking the first found result.
In our old sample response, there was just one matching Vet360 ID.
In the new response there are multiple Vet360 IDs.
Is just taking the first the best idea still?  Should all of them be returned by the parser instead (like `mhv_ids`, `vha_facility_ids`, etc)? 

I added check marks in the code below to indicate matching rows.

Old response:
```
<subject1 typeCode="SBJ">
    <patient classCode="PAT">
        <id extension="1008714701V416111^NI^200M^USVHA^P" root="2.16.840.1.113883.4.349"/>
        √ <id extension="123456789^PI^200VETS^USDVA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="796122306^PI^200BRLS^USVBA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="9100792239^PI^200CORP^USVBA^A" root="2.16.840.1.113883.4.349"/>
```

New response:
```
<subject1 typeCode="SBJ">
    <patient classCode="PAT">
        <id extension="1008787551V609092^NI^200M^USVHA^P" root="2.16.840.1.113883.4.349"/>
        <id extension="768598574^PI^200BRLS^USVBA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="0000001008787551V609092000000^PI^200ESR^USVHA^A" root="2.16.840.1.113883.4.349"/>
        √ <id extension="1000^PI^200VETS^USDVA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="201801104^PI^523^USVHA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="22710152^PI^585^USVHA^A" root="2.16.840.1.113883.4.349"/>
        √ <id extension="2011801312^PI^200VETS^USDVA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="32364231^PI^200CORP^USVBA^A" root="2.16.840.1.113883.4.349"/>
        √ <id extension="201808031^PI^200VETS^USDVA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="201804032^PI^200VETS^USDVA^H" root="2.16.840.1.113883.4.349"/>
        <id extension="7174051^PI^608^USVHA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="0000001008787550V443247000000^PI^200ESR^USVHA^H" root="2.16.840.1.113883.4.349"/>
        <id extension="875637483^PI^200BRLS^USVBA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="32364195^PI^200CORP^USVBA^A" root="2.16.840.1.113883.4.349"/>
        √ <id extension="80^PI^200VETS^USDVA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="201806031^PI^500^USVHA^A" root="2.16.840.1.113883.4.349"/>
        <id extension="201806032^PI^500^USVHA^A" root="2.16.840.1.113883.4.349"/>
```

Reply:
oof, I don't think we're equipped to handle multiple vets360 id's properly.  I think this version upgrade was supposed to give us additional status codes (id's that end in `^L`).  Maybe we need to reach out to the MVI or Vets360 teams to see how to properly handle.

https://dsva.slack.com/archives/CK1FA11H8/p1565638882180700

Reply:
Confirmed with Vet360 that a user should only have one active (`A`) id.
MVI changed the in this `dev` environment for this user to reflect this.
We _shouldn't_ run into this issue in production.